lib/opencensus/stats/measurement.rb

```diff
@@ -0,0 +1,11 @@
+# frozen_string_literal: true
+
+
+module OpenCensus
+  module Stats
+    # Struct that holds measurement value
+    # @attr [Measure] Measure details
+    # @attr [Integer, Float] Value of the measurement
+    Measurement = Struct.new(:measure, :value)
```

Comment:
The [proto](https://github.com/census-instrumentation/opencensus-proto/blob/master/src/opencensus/proto/stats/v1/stats.proto) includes a `tags` field and a `time` field which seem to be missing here.

Also, I suspect `Measurement` should be immutable (that is, it shouldn't be a struct).

Reply:
Added time and tags. Reface struct with class.